Makefile.work

```diff
@@ -172,7 +177,8 @@ DOCKER_BASE_BUILD = docker build --no-cache \
 		    -t $(SLAVE_BASE_IMAGE):$(SLAVE_BASE_TAG) \
 		    --build-arg http_proxy=$(http_proxy) \
 		    --build-arg https_proxy=$(https_proxy) \
-		    $(SLAVE_DIR)
+		    $(SLAVE_DIR); \
+		    scripts/collect_docker_version_files.sh $(SLAVE_BASE_IMAGE):$(SLAVE_BASE_TAG) target
```

Comment:
>scripts/collect_docker_version_files.sh $(SLAVE_BASE_IMAGE):$(SLAVE_BASE_TAG) target [](start = 6, length = 84)

In concept, it is not relevant with docker build. Could you find a better place?

Reply:
It maybe the best place to collect the version info after the docker image built inside the slave.

Reply:
I think you can remove this line since it is already after each `DOCKER_BASE_BUILD`.

---
In reply to: [526570002](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r526570002) [](ancestors = 526570002)

Reply:
Yes, it should be removed.

---
In reply to: [534558624](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r534558624) [](ancestors = 534558624,526570002)